source/jsonnet/northern-ireland/individual/blocks/employment/hours_worked.jsonnet

```diff
@@ -1,8 +1,38 @@
 local placeholders = import '../../../lib/placeholders.libsonnet';
 local rules = import 'rules.libsonnet';
 
-local question(title) = {
-  title: title,
+local questionTitle(isProxy, isEmployed) = (
```

Comment:
I find this change hard to read, particularly the else, else. Any reason I am missing why it has been changed? I was looking for 'if the' to be added